Assets/Scripts/Game/Guilds/ThievesGuild.cs

```diff
@@ -76,6 +77,16 @@ public class ThievesGuild : Guild
 
         #region Guild Membership and Faction
 
+        public ThievesGuild()
+        {
+            RegisterEvents();
```

Comment:
Same as for the DarkBrotherhood class, not needed since events are registered when joining the guild or when loading the game.